circus/tests/test_util.py

```diff
@@ -92,8 +92,11 @@ def test_convert_opt(self):
         self.assertEqual(util.convert_opt('test', 1), '1')
 
     def test_bytes2human(self):
-        self.assertEqual(bytes2human(10000), '9K')
-        self.assertEqual(bytes2human(100001221), '95M')
+        self.assertEqual(bytes2human(100), '100B')
+        self.assertEqual(bytes2human(10000), '9.76K')
```

Comment:
Ideally this should be `9.76 KiB`, but I'm probably being too pedantic here ;) .

Reply:
KiB notation is kind of controversial, despite being a standard. It is not as widely adopted in my opinion as the one used right now.

Anyway, if you think this is the right way I can do that in a separate PR.

Reply:
Let's keep `K` for simplicity for now.